### DISCUSSION TITLE
Duda sobre practica 1

### DISCUSSION THREAD

**Discussion by Darry96** [Launch-X-Latam/MisionFrontEnd#199] [Q&A]
He estado haciendo mi diseños para la pagina pero al revisar las demas aportaciones de los demas compañeros sale veo que las crearon basado en Desktop. Mi pregunta que querra decir la parte donde dice que sea responsivo en movil?? Es que yo cree mi diseño en movil por que entendi que tenia que ser asi :(

**Comment by AtenasHcrz** [comment]
Hola! en el primer párrafo se menciona:
Es un despacho de abogados que quiere automatizar las demandas de sus clientes, esto lo harán a través de una página web llenando un formulario.

y que sea responsivo se refiere a que pueda ser visualizable también en móvil c:
